radio/src/gui/colorlcd/model_setup.cpp

```diff
@@ -1033,7 +1033,8 @@ void ModelSetupPage::build(FormWindow * window)
 
   // Model name
   new StaticText(window, grid.getLabelSlot(), STR_MODELNAME);
-  auto text = new RadioTextEdit(window, grid.getFieldSlot(), g_model.header.name, sizeof(g_model.header.name));
+  auto text = new RadioTextEdit(window, grid.getFieldSlot(), g_model.header.name, sizeof(g_model.header.name),
+          0, "_-.,:;<=>");
```

Comment:
Can we please have these in some kind of definitions? Ideally, some `const static`.

Reply:
we of course could
I would need some suggestion though
- where/which file should it be placed?
- recommendation for name of it?

Reply:
Very good question! Frankly, I have no clue for now, inside the same file would be ok for now, but we will need some restructuring soon, this code is a mess :-D

Reply:
that's why I just put it here for now ... for as long as it is not needed/required elsewhere it's not so much of  harm 
(and efficiency wise it doesn't matter anyways, the compiler does its optimization job)

Reply:
ok, so did something
have to go now however, day work

Reply:
Thx! Looks ok to me!

Reply:
great, so I mark it as resolved